src/rpc/blockchain.cpp

```diff
@@ -1714,7 +1714,10 @@ UniValue getspecialtxes(const JSONRPCRequest& request)
     UniValue result(UniValue::VARR);
     for(const auto& tx : block.vtx)
     {
-        if (tx->nVersion != 3 || tx->nType == TRANSACTION_NORMAL || (nTxType != -1 && tx->nType != nTxType)) continue;
+        if (tx->nVersion != 3 || tx->nType == TRANSACTION_NORMAL // ensure it's in fact a special tx
```

Comment:
Hmm, what if a user wants to use this RPC to filter for normal TXs? Maybe remove the `|| tx->nType == TRANSACTION_NORMAL` part?

Reply:
Hmm.. but is normal tx a special tx? :D